feature-utils/poly-look/src/utils/smoothScroll.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * A function to get smooth scrolling behaviour on all browsers (scroll-behaviour:smooth doesn't work in safari)
+ * @param {string} childId - Dom element with ID that is being scrolled to
+ * @param {string} [parentId] - Dom element with ID that is scrollable and parent of child element
+ * @param {string[]} [topNavigationElementIds] - Array of Dom ID's that sit on top of the parent element (usually navigation elements)
+ * @param {Object} [options] - Options regarding the scrolling behaviour
+ * @param {string} [options.speed = 10] - Speed of the scrolling (in ms; smaller = faster)
+ * @param {string} [options.step = 10] - Steps size of the scrolling (smaller = smoother but slower)
+ */
+export function SmoothScroll(
```

Comment:
Nit: The upper camel case naming suggests it's either a class, or a React component - but it's neither, it's a function. So typically that'd be lower camel case and a verb in the name - pretty much how it was called, `scrollSmoothly`. I think `smoothScroll` is fine though, but `SmoothScroll` seems pretty confusing.